packages/xdg_directories/example/README.md

```diff
@@ -0,0 +1,20 @@
+# XDG Directories Demo
+
+## Description
+
+This is a simple demo of the XDG directories.  It is a flutter app that
+uses the xdg_directories package to read xdg directory names and display them in a
+list.
+
+## Run
```

Comment:
We don't need to explain Flutter basics, or include a screenshot, in the example's README.

Reply:
Got it.